globus_portal_framework/templatetags/random_numbers.py

```diff
@@ -0,0 +1,11 @@
+import random
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag
+def random_int(a, b=None):
+    if b is None:
+        a, b = 0, a
+    return random.randint(a, b)
```

Comment:
What is this for?

Reply:
They had in the USGS project, and I assumed it was getting called on the HTML page. But you are right, looks like its doing nothing on the `search-results.html` page. Ill remove the file and the reference on that page.